text/0000-async_await.md

```diff
@@ -69,14 +70,14 @@ async fn function(argument: &str) -> usize {
 }
 ```
 
-Async functions work differently from normal functions. When an async function
-is called, it does not enter the body immediately. Instead, it evaluates to an
-anonymous type which implements `Future`. As that future is polled, the
-function is evaluated up to the next `await` or return point inside of it (see
-the await syntax section next).
+`async` functions work differently than normal functions. When an `async`
```

Comment:
"different from" is correct

Reply:
Interesting. Didn't know that. (I'm from Germany ;)